test/io/pathfinder/test/ClusterSpec.java

```diff
@@ -0,0 +1,299 @@
+package io.pathfinder.test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import io.pathfinder.models.Commodity;
+import io.pathfinder.models.Vehicle;
+import play.libs.Json;
+import play.mvc.Result;
+import play.test.Helpers;
+import play.mvc.Http.RequestBuilder;
+import play.test.FakeApplication;
+import play.core.j.JavaResultExtractor;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Before;
+import static org.junit.Assert.*;
```

Comment:
IDE auto converts five or more imports to wildcards. Although, it is a static import. So it is just importing all the static methods in the Assert class. They all come from the same location.